### PR TITLE
Add instructions for setting up CORS (GEOS-7919)

### PR DESCRIPTION
**Pull request by jodygarnett**
See: https://osgeo-org.atlassian.net/browse/GEOS-7919

### CODE REVIEW

File: doc/en/user/source/production/container.rst [hunk 1]
```diff
@@ -52,4 +52,29 @@ For more information about JVM configuration, see the article `Performance tunin
...
       uintx MaxHeapSize       := 4294967296    {product}
     
-   The above results (from a 16 GB laptop) amount to initial heap size of 256m, and a max heap size of around 4 GB (or around 1/4 of system memory).
\ No newline at end of file
+   The above results (from a 16 GB laptop) amount to initial heap size of 256m, and a max heap size of around 4 GB (or around 1/4 of system memory).
+   
+Enable CORS
+-----------
+
+The standalone distributions of GeoServer use an application server include the Jetty application server. Enable Cross-Origin Resource Sharing (CORS) to allow JavaScript applications outside of your own domain to use GeoServer.
```
Comment (bradh) [inline doc/en/user/source/production/container.rst]: Suggest "The standalone distributions of GeoServer use an application server." or "The standalone distributions of GeoServer include the Jetty application server." in place of the first sentence, which doesn't make sense to me as written.

File: doc/en/user/source/production/container.rst [hunk 2]
```diff
@@ -52,4 +52,29 @@ For more information about JVM configuration, see the article `Performance tunin
...
+    ...
+    <filter-mapping>
+        <filter-name>cross-origin</filter-name>
+        <url-pattern>/*</url-pattern>
+    </filter-mapping>
+    ...
+   </web-app>
+   ````
+
+# This requires "jetty/lib/jetty-servlets.jar" which can be downloaded seperately.
```
Comment (bradh) [inline doc/en/user/source/production/container.rst]: seperately -> separately

File: doc/en/user/source/production/container.rst [hunk 3]
```diff
@@ -52,4 +52,29 @@ For more information about JVM configuration, see the article `Performance tunin
...
-   The above results (from a 16 GB laptop) amount to initial heap size of 256m, and a max heap size of around 4 GB (or around 1/4 of system memory).
\ No newline at end of file
+   The above results (from a 16 GB laptop) amount to initial heap size of 256m, and a max heap size of around 4 GB (or around 1/4 of system memory).
+   
+Enable CORS
+-----------
+
+The standalone distributions of GeoServer use an application server include the Jetty application server. Enable Cross-Origin Resource Sharing (CORS) to allow JavaScript applications outside of your own domain to use GeoServer.
+
+# Add the following configuration to :file:`webapps/geoserver/WEB-INF/web.xml`:
```
Comment (bradh) [inline doc/en/user/source/production/container.rst]: Could be worth adding an external link to http://www.eclipse.org/jetty/documentation along the lines of "For more information on what this does and other options see http://www.eclipse.org/jetty/documentation"

**Comment by jodygarnett** [comment]
This has now been fixed by @jrbeckwith (https://github.com/boundlessgeo/geoserver/pull/7).

**Comment by aaime** [comment]
@jodygarnett your comment made it look like it was fixed only in the opengeo suite, but checking the commits I see it in the master branch too. It's unclear if @bradh comments were considered, I did not check.

**Comment by jodygarnett** [comment]
Thanks will double back and add in the requested documentation link above.

**Comment by jodygarnett** [comment]
Addressed as per https://github.com/geoserver/geoserver/pull/2069

**Pull request closed without merging**